simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -54,15 +56,42 @@
  */
 public class PushGateway {
 
-  private final String address;
+  // Visible for testing
```

Comment:
The comment is good. Please add a full stop.

For clarity I'd have a blank line after the variable, and explicitly make it `protected`.